recipes-ni/ni-grpc-device/ni-grpc-device_1.1.bb

```diff
@@ -8,7 +8,13 @@ LIC_FILES_CHKSUM = "\
 	file://${PN}-server/ThirdPartyNotices.txt;md5=6def9ca42f278e76d89243699fae2b67 \
 "
 
+inherit ptest
```

Comment:
STYLE: better to put this inherit line below the class-generic variables (see [this guide](https://www.openembedded.org/wiki/Styleguide#Ordering_and_grouping)). You could also group it with the `RDEPENDS_${PN}-ptest` and `do_install_ptest_append ()` lines.

Reply:
As you wish.